spec/models/post_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) { User.new(name: 'Khusniddin Ismoilov', photo: 'https://picsum.photos/400', bio: 'I am a full-stack software engineer.') }
+  subject { Post.new(title: 'Congrats', text: 'Congratulations', author: user) }
+  before { subject.save }
+
+  it 'has title' do
+    expect(subject.title).to eq('Congrats')
+  end
+
+  it 'has text' do
+    expect(subject.text).to eq('Congratulations')
+  end
+
+  it 'title has a maximum length of 250 characters' do
+    expect(subject.title.length).to be <= 250
+  end
```

Comment:
- [ ] Something similar happens here, since you're working on validations, please focus on whether or not these cases are valid, and consider both of them, I see that you added the not valid cases at the end! 🤓 🔧